server/sendEmail.js

```diff
@@ -0,0 +1,13 @@
+require("dotenv").config();
+const sgMail=require('@sendgrid/mail');
+sgMail.setApiKey('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');
```

Comment:
You should NEVER expose API keys in your code

Reply:
Resolved and stored the api key in apikey.js file